src/styles/index.scss

```diff
@@ -1,3 +1,203 @@
+$star-number: 5;
+$color-minor: #616070;
+$color-accent: #00acdc;
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  background-color: #fff;
+  color: #060b35;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  height: 408px;
+
+  position: relative;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  font-weight: 400;
+  letter-spacing: 0;
+
+  &__image-container {
+    position: absolute;
+    top: 32px;
+    left: 19px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    padding: 0;
+
+    width: 166px;
+    height: 36px;
+
+    position: absolute;
+    top: 206px;
+    left: 16px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    margin: 0;
+    padding: 0;
+
+    width: 98px;
+    height: 14px;
+
+    position: absolute;
+    top: 246px;
+    left: 17px;
+
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $color-minor;
+  }
+
+  &__reviews {
+    width: 166px;
+    height: 16px;
```

Comment:
Made some changes.
But I still don't know how to avoid using static width in .card__reviews and .card__price classes. 
What can you recommend in this case?